cmd/utils/flags.go

```diff
@@ -296,6 +296,10 @@ var (
 		Name:  "exec",
 		Usage: "Execute JavaScript statement (only in combination with console/attach)",
 	}
+	PreLoadJSFlag = cli.StringFlag{
+		Name:  "preload",
+		Usage: "Comma separated list with JS files to pre-load into the console",
```

Comment:
Nitpick, but could we formulate as `Comma separated list of JavaScript files to preload into the console`? AFAIK we always typed out `JavaScript` in full and it makes it a bit more obvious to people.

Also please rename all `pre-load` to `preload` http://www.merriam-webster.com/dictionary/preload